File: common/src/main/java/com/reservation/common/support/imageuploader/MockImageUploader.java
package com.reservation.common.support.imageuploader;

import org.springframework.stereotype.Component;
import org.springframework.web.multipart.MultipartFile;

import lombok.extern.log4j.Log4j2;

@Log4j2
@Component
public class MockImageUploader implements ImageUploader {
	@Override
	public String upload(MultipartFile file, String url) {
		return url;
	}

	@Override
	public void delete(String url) {
		log.info("이미지 삭제됨 (mock): {}", url);
	}
}

File: file-upload/src/main/kotlin/com/reservation/fileupload/imageupload/service/MockImageUploader.kt
package com.reservation.fileupload.imageupload.service

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.stereotype.Component
import org.springframework.web.multipart.MultipartFile

@Component
class MockImageUploader : ImageUploader {
    private val log: Logger = LoggerFactory.getLogger(MockImageUploader::class.java)

    override fun upload(file: MultipartFile, url: String): String {
        return url
    }

    override fun delete(url: String) {
        log.info("이미지 삭제됨 (mock): {}", url)
    }
}
